presto-native-execution/presto_cpp/main/operators/tests/BroadcastTest.cpp

```diff
@@ -143,4 +221,125 @@ TEST_F(BroadcastTest, broadcastWrite) {
   runBroadcastTest({data});
 }
 
+TEST_F(BroadcastTest, endToEndWithMultipleWriteNodes) {
+  auto data1 = vectorMaker_.rowVector({
+      makeFlatVector<int32_t>({1, 2, 3, 4, 5, 6}),
+      makeFlatVector<int64_t>({10, 20, 30, 40, 50, 60}),
+  });
+  auto data2 = vectorMaker_.rowVector({
+      makeFlatVector<int32_t>({11, 21, 31, 41, 51, 61}),
+      makeFlatVector<int64_t>({102, 203, 304, 405, 506, 607}),
+  });
+  auto basePath = exec::test::TempDirectoryPath::create()->path;
+
+  // Execute write 1
+  auto [taskCursor1, results1] = executeBroadcastWrite({data1}, basePath);
```

Comment:
These variable names are an anti-pattern. Consider writing a loop and using std::vectors instead.

Reply:
Done